src/app/replayer/replayer.ml

```diff
@@ -352,7 +404,16 @@ let apply_combined_fee_transfer ~logger ~pool ~ledger
   in
   match undo_or_error with
   | Ok _undo ->
-      Deferred.unit
+      (* in Transaction_log.process_transfer_fee, when the fee transfer has two components,
```

Comment:
I think -- although perhaps @deepthiskumar can confirm -- that we will never see a fee transfer with the same receiver in both components because of how we synthesise them from the pre-diff in a block.

Reply:
[Confirming my own suspicion](https://github.com/MinaProtocol/mina/blob/5cc4ace7b04345d2ac63ae730b0c4cb1d4f4103a/src/lib/staged_ledger/pre_diff_info.ml#L239). (Apologies for the unnecessary ping.)